ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -28,7 +28,16 @@ define(
              */
             initialize: function () {
                 this._super();
-                this._loadApi();
+
+                this.attachFocusEvent();
+            },
+
+            attachFocusEvent: function () {
+                const self = this, $parentForm = $('#' + this.getReCaptchaId() + '-container').parents('form');
+
+                $parentForm.find('input, select').on('focus', function () {
```

Comment:
Just wonder if we can change it to "one" since we don't need to load it multiple times
```suggestion
                $parentForm.find('input, select').one('focus', function () {
```